lib/beaker/host/unix/exec.rb

```diff
@@ -142,7 +142,7 @@ def delete_env_var key, val
   #  host.get_env_var('path')
   def get_env_var key
     key = key.to_s
-    exec(Beaker::Command.new("env | grep #{key}"), :accept_all_exit_codes => true).stdout.chomp
+    exec(Beaker::Command.new("env | grep ^#{key}="), :accept_all_exit_codes => true).stdout.chomp
```

Comment:
Sooo... having never looked at this code before, this appears to be doing a command execution with an arbitrary unescaped+unquoted parameter embedded.

In the original, if I call `get_env_key("whatever ; rm -rf /")`, or in the modified version, if I call `get_env_key("whatever ; rm -rf / ; # ")` then presumably the caller can execute whatever nefarious command they choose.

I don't know all the places where this is called, and what the threat model might be, but looks kinda wide open.  Am I missing something?

cc @puppetlabs/beaker